### PR TITLE
[TESTS] Added instructions for running batch test in README

### PR DESCRIPTION
**Pull request by susanxhuynh**

### CODE REVIEW

File: scale-tests/README.md [hunk 1]
```diff
@@ -42,29 +42,95 @@ This will upload the "tests assembly" JAR to S3 and output its public object URL
...
 your-machine $ git clone git@github.com:mesosphere/spark-build.git
 your-machine $ cd spark-build
-your-machine $ docker run -it --net=host -v (pwd):/spark-build -v </path/to/soak/key>:/ssh/key mesosphere/spark-build:latest
+your-machine $ docker run -it --net=host -v $(pwd):/spark-build -v </path/to/soak/key>:/ssh/key mesosphere/spark-build:latest
 docker-container # cd /spark-build
 docker-container # pip3 install -r tests/requirements.txt
 docker-container # export PYTHONPATH=$(pwd)/testing:$(pwd)/spark-testing
 docker-container # dcos cluster setup --insecure --username=<username> --password=<password> <cluster-url>
 docker-container # eval "$(ssh-agent)"
+docker-container # chmod 600 ccm.pem
```
Comment (elezar) [inline scale-tests/README.md]: Does this makes sense from the context of the docker container? The file `ccm.pem` does not exist, and is added as `/ssh/key`. One could change the mount argument in the docker run command to `-v </path/to/soak/key>:/ssh/key:ro` to indicate that this should be readonly.

File: scale-tests/README.md [hunk 2]
```diff
@@ -42,29 +42,95 @@ This will upload the "tests assembly" JAR to S3 and output its public object URL
...
+SUBMITS_PER_MIN=1
+
+./scale-tests/batch_test.py \
+  $DISPATCHERS_OUTPUT_FILE \
+  --submits-per-min $SUBMITS_PER_MIN
+```
+
+#### 3. Alternatively, deploy a Marathon app which launches batch jobs from within the cluster [optional]
+
+First, upload the DISPATCHERS_OUTPUT_FILE created above somewhere that is accessible from the cluster -
```
Comment (elezar) [inline scale-tests/README.md]: Let's add the command here (see the example for the JAR in the streaming app).

File: scale-tests/README.md [hunk 3]
```diff
@@ -42,29 +42,95 @@ This will upload the "tests assembly" JAR to S3 and output its public object URL
...
+S3, for example.
+
+Then run the following, filling in values for username, password, and S3 bucket:
+
+```bash
+DCOS_USERNAME=<username>
+DCOS_PASSWORD=<password>
+S3_BUCKET=<bucket>
+DISPATCHERS_OUTPUT_FILE_URL="https://${S3_BUCKET}.s3.amazonaws.com/${DISPATCHERS_OUTPUT_FILE}"
```
Comment (elezar) [inline scale-tests/README.md]: Should we avoid putting this in the root of the bucket?

File: scale-tests/README.md (hunk 2 above)
Comment (susanxhuynh) [reply scale-tests/README.md]: Done

File: scale-tests/README.md (hunk 3 above)
Comment (susanxhuynh) [reply scale-tests/README.md]: Added a folder

**Pull request merged**